src/components/DatePickerContainer/DatePickerContainer.css

```diff
@@ -0,0 +1,11 @@
+.infoPanel {
```

Comment:
Class names should use hyphen-delimited-syntax instead of camelCase ([source](https://csswizardry.com/2010/12/css-camel-case-seriously-sucks/))